docs/core/tools/csproj.md

```diff
@@ -37,7 +37,7 @@ Since `Microsoft.NETCore.App` or `NetStandard.Library` metapackages are implicit
 * If you need a specific version of the runtime when targeting .NET Core, you should use the `<RuntimeFrameworkVersion>` property in your project (for example, `1.0.4`) instead of referencing the metapackage.
     * This might happen if you are using [self-contained deployments](../deploying/index.md#self-contained-deployments-scd) and you need a specific patch version of 1.0.0 LTS runtime, for example.
 * If you need a specific version of the `NetStandard.Library` metapackage when targeting .NET Standard, you can use the `<NetStandardImplicitPackageVersion>` property and set the version you need.
-* Do not explicitly add or update references to either the `Microsoft.NETCore.App` or `NetStandard.Library` metapackage in .NET Framework projects. If any version of `NetStandard.Library` should really be needed when using a .NET Standard based NuGet package, NuGet will automatically install the needed version.
+* Don't explicitly add or update references to either the `Microsoft.NETCore.App` or `NetStandard.Library` metapackage in .NET Framework projects. If any version of `NetStandard.Library` is needed when using a .NET Standard-based NuGet package, NuGet automatically installs the needed version.
```

Comment:
suggestion: install the needed version -> installs that version

Reply:
much better, changed it

Reply:
Cool. I'll merge when build finishes. Thanks @dasMulli and apologies again on the delay. Feel free to ping me when I'm taking more than a few days to respond.